rnd/autogpt_builder/src/components/CustomNode.tsx

```diff
@@ -166,6 +166,17 @@ const CustomNode: FC<NodeProps> = ({ data, id }) => {
     }
   };
 
+  const getStatusValue = (statusData: any) => {
```

Comment:
you don't need this anymore? 

I think using this still works if you wanted to

Reply:
yeah this is not needed any more it seems so removed it